app/UI/edit-profile.tsx

```diff
@@ -185,6 +183,37 @@ export class EditProfile extends Component<Props, State> {
         this.newFieldValue.current.value = "";
     };
 
+    uploadImage = (e: h.JSX.TargetedEvent<HTMLInputElement>) => {
```

Comment:
Just a code reorg right?

Reply:
It was copied from `editor.tsx` and unnecessary branches were deleted. Maybe we will abstract this part out when we need to upload images again in the future.